proto/regen/data/v1alpha2/types.proto

```diff
@@ -0,0 +1,110 @@
+    //
+    // ## **General Rules**
+    //
+    // - only the WGS 84 reference coordinate system is supported and all coordinates should be assumed to be in WGS 84
+    // - only 2D and 3D coordinates are supported, coordinates with linear referencing systems or an “M” component are unsupported
+    //
+    // ## **`POLYGON` Rules**
+    //
+    // - exterior linear rings must be defined in counterclockwise direction
+    // - interior linear rings must be defined in clockwise direction
+    // - where there is more than one ring, the first MUST be the exterior ring and other rings MUST be linear rings - this means that the area of a polygon MUST be continuous and unbroken. To represent multiple non-contiguous shapes, a `MULTIPOLYGON` should be used instead. Ex: in the case where inside of a hole in a polygon, there is an island that we want to include. Because the island is disjoint, this shape must be represented as a `MULTIPOLYGON` instead of a `POLYGON` with a hole and an island in the hole
+    // - the first and last point in a linear ring must be identical
+    // - *canonical coordinate ordering rule*: the first (and last) point in a linear ring must be the point with the minimum X value in the ring. In the event where there is more than one point in a ring with the minimum X value, the first point must be the one with the minimum Y value. For 3D geometries, if there is is more than one point with the minimum X and Y values, the one with the minimum Z value should be the first.
+    // - if there is more than one interior linear ring, those rings should be ordered using the above *canonical coordinate ordering rule*, i.e. the ring with the minimum X coordinate comes first, etc.
+    //
+    // ## `**MULTIPOLYGON` Rules**
+    //
+    // - each polygon should conform to the above `POLYGON` rules
+    // - if there is more than one polygon, those polygons should be ordered using the above *canonical coordinate ordering rule*, i.e. the polygon with the minimum X coordinate comes first, etc.
+    ID_TYPE_GEOGRAPHY = 2;
+}
+
+
+// MediaType defines MIME media types to be used with ID_TYPE_RAW_UNSPECIFIED.
+enum MediaType {
+    MEDIA_TYPE_UNSPECIFIED = 0;
+
+    // basic formats
+    MEDIA_TYPE_TEXT_PLAIN = 1;
+    MEDIA_TYPE_JSON = 2;
+    MEDIA_TYPE_CSV = 3;
+    MEDIA_TYPE_XML = 4;
+    MEDIA_TYPE_PROTOBUF_ANY = 5;
+    MEDIA_TYPE_PDF = 6;
+
+    // images
+    MEDIA_TYPE_TIFF = 16;
+    MEDIA_TYPE_JPG = 17;
+    MEDIA_TYPE_PNG = 18;
+    MEDIA_TYPE_SVG = 19;
+    MEDIA_TYPE_WEBP = 20;
+    MEDIA_TYPE_AVIF = 21;
+    MEDIA_TYPE_GIF = 22;
+    MEDIA_TYPE_APNG = 23;
+
+    // audio-visual media containers
+    MEDIA_TYPE_MPEG = 32;
+    MEDIA_TYPE_MP4 = 33;
+    MEDIA_TYPE_WEBM = 34;
+    MEDIA_TYPE_OGG = 35;
+
+    // geospatial raster formats
+    // TODO
+
+    // geospatial vector formats
+    // TODO
+}
+
+enum GraphCanonicalizationAlgorithm {
+    GRAPH_CANONICALIZATION_ALGORITHM_URDNA2015_UNSPECIFIED = 0;
```

Comment:
Shouldn't we have this instead?
```
GRAPH_CANONICALIZATION_ALGORITHM_UNSPECIFIED = 0;
GRAPH_CANONICALIZATION_ALGORITHM_URDNA2015 = 1;
```

Reply:
I was going for a sensible default rather than 0 being an error. I don't really have a strong opinion

Reply:
One argument for having `UNSPECIFIED` be an error is that defaults may change over time. Thinking about the conversation you bring up [here](https://github.com/regen-network/regen-ledger/discussions/222), it seems like we're still evaluating what a sensible default may be?

Reply:
> Thinking more about user-facing features, what do we wanna provide in addition to anchor/sign/store to actually create the `ContentID`s in the first place? I guess that for RDF, we can move to [this discussion](https://github.com/regen-network/regen-ledger/discussions/222), what about the other types (although for those types, it seems less complex and computationally intensive)?

I'm thinking that signing is mainly relevant to RDF data and not raw/graph data. We generally just care about integrity and availability of that data. But all structure claims that would be signed would be RDF... So maybe we limit sign with that in mind? Also most raw data like PDFs, images, etc. would be too big to store on-chain (I'm thinking we'd have a 1kb on-chain storage limit or something).

Reply:
I'm in favor of liminting signing capabilities to graph data.  Signing always represents attesting a claim, and in RDF land this just means claims are always represented as triples. If someone wants to attest to the veracity of a PDF, they might as well attest to the connection of that PDF to a given project or credit class, which would be stored onchain as an rdf graph.